server/controllers/events/testfixtures/test-repos/simple-yaml/exp-output-autoplan.txt

```diff
@@ -39,6 +39,7 @@ postplan
 </details>
 Plan: 1 to add, 0 to change, 0 to destroy.
 
+
```

Comment:
why these changes?

Reply:
in `server/events/templates/planSuccessWrapped.tmpl` I added a newline at the end of the template. Some templates have a new line and some don't. Whenever I encounter templates without a newline I just add it and update e2e templates.